### PR TITLE
Portflio task

### PR DESCRIPTION
**Pull request by Athaliah8**
##In This repo

[ ] Added Linters
[ ] Added Html and Css file
[ ] Added gitflow
[ ]Added linter error file
[ ]Added images folder

### CODE REVIEW

**Review by ha-manel — COMMENTED** [review]
# STATUS: INVALID 🔴 
Hi @Athaliah8 
I'm sure you put a lot of effort into this project but this PR is invalid because you have linter errors😓 

image

Kindly fix the issue and submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Images/index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>Athaliah Portflio SetUp</title>
+ <link rel="stylesheet" href="style.css">
```
Comment (Nyame-Wolf) [inline Images/index.html]: Kindly fix the path to this css file. Currently they do not show the styles you implemented. 
This is your project currently.
Screenshot 2022-10-28 at 17 02 10


This is the figma template
Screenshot 2022-10-28 at 11 23 07

I suggest you either move the index.html to the root folder or fix the path.

File: Images/index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    <!--ToolBar-->
+   
+    <header>
+ <div class="container " >
+    
+    <div class="wrapper"> 
+        <nav id="nav">
+            
+            <div class="myLogo">      
+<image src="./logo.svg" width="Hug (76px)" >
```
Comment (Nyame-Wolf) [inline Images/index.html]: This should be a link tag `<a>` 
This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

File: Images/index.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+<h1>I'm Athaliah Glad to see you!</h1>  
+<p>
+I'm a software developer ! I can help you build a product , 
+<br>feature or website Look through some of my work and experience!<br>
+If you like what you see and have a project you need coded, don't hestiate to contact me.
+
+</p>
+<footer>
+    
+<h2>LET'S CONNECT</h2>
```
Comment (Nyame-Wolf) [inline Images/index.html]: Please fix this to be a link `<a>`. By its a color, you can see that its blue. This should redirect people visiting your page to a contact form or your socials.

File: Images/index.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+<div class="container" onclick="myFunction(this)">
+    <div class="bar1"></div>
+    <div class="bar2"></div>
+    <div class="bar3"></div>
+
+<!--Main-Content-->
+  </div>
+</div>
+ </div>
+ <section id="Main_Content">
```
Comment (Nyame-Wolf) [inline Images/index.html]: This section should have background images as shown in the figma design.
Screenshot 2022-10-28 at 11 23 07

 Its a requirement to pass this milestone.

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @Athaliah8  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Linters are passing.
- Pr description is professional

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 5]
```diff
@@ -1,72 +1,131 @@
...
+      margin: auto;
+      background-color: white;
+      height: 50vh;
+      color: #344563;
+      border-radius: 10px;
+    }
+
+    .topnav {
+      overflow: hidden;
+      background-color: lightblue;
```
Comment (emmiiorji) [inline index.html]: As requested by the previous reviewer, kindly avoid the use of inline styling. External styles make for more reusable styles.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+    290px 509px,
+    240px 507px,
+    357px 392px,
+    106px 497px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 600px;
+}
```
Comment (emmiiorji) [inline style.css]: Good job at adding the background image as requested by the previous reviewer. Kindly modify the section to follow the design template by sticking to font sizes and colours.

Your design:
image

Figma design:
image

Note that:

- The text of the intro text does not overlap the background image
- The headline text is bolder than you have it
- There are some missing social icons in your design
- The LET'S CONNECT text together with the social icons are vertically aligned with the intro text and headline text.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Athaliah8, 

# Status: Changes Required ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Almost there

 ## Good Highlights✔

- Linter checks are passing
- Good commit messages
- Adherence to GitHub flow


 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @Athaliah8  👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**